macOS/LocalPackages/FeatureFlags/Sources/FeatureFlags/FeatureFlag.swift

```diff
@@ -340,7 +340,8 @@ extension FeatureFlag: FeatureFlagDescribing {
         case .subscriptionRebranding:
             return .remoteReleasable(.subfeature(PrivacyProSubfeature.subscriptionRebranding))
         case .newFeedbackForm:
-            return .disabled
+            return .enabled
+//            return .remoteReleasable(.feature(.feedbackForm))
```

Comment:
This is enabled for making it easier to test for this PR review. Before merging I will remove the enabled in favor of the .remoteReleasable(.feature(.feedbackForm)) line.